.github/workflows/gluon_build_m1.yml

```diff
@@ -0,0 +1,98 @@
+# Builds GraalVM for Linux, Windows and macOS, wiht JDK 11 and 17,
+# and replaces java static libraries on Linux 17 after building them with gcc 9.3
+
+name: Gluon Release for M1
+
+on:
+  push:
+    tags:
+      - '*'
+
+env:
+  LANG: en_US.UTF-8
+
+jobs:
+  determine-version:
+    runs-on: ubuntu-20.04
+    outputs:
+      simpleTag: ${{ steps.fetchSimpleTag.outputs.tag }}
+      version: ${{ steps.fetchVersion.outputs.version }}
+    steps:
+    - id: fetchSimpleTag
+      run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+    - id: fetchVersion
+      run: echo ::set-output name=version::$(echo $GITHUB_REF | sed -E "s/.*(gluon-.*-(dev|Final)).*/\1/g")
+
+  build-graalvm-darwin-m1:
```

Comment:
Any reason why this is not inside `gluon_release.yml` together with the other platforms?